cairo/tests/ethereum/cancun/vm/instructions/test_block.py

```diff
@@ -152,3 +155,19 @@ def test_chain_id(self, cairo_run, evm: Evm):
 
         chain_id(evm)
         assert evm == cairo_result
+
+
+class TestUtils:
+    @given(logs=..., new_logs=...)
+    def test_append_logs(
+        self, cairo_run, logs: Tuple[Log, ...], new_logs: Tuple[Log, ...]
```

Comment:
There is a Tuple registered strategy but not a Log registered strategy.
Data field of Logs can hence be huge and fuzzing could "focus" on this parameter across the codebase. 
I think it makes sense to create a log strategy and limit a little bit this field

Reply:
not mandatory to it in this PR 
An issue could be created if you prefer

Reply:
done